src/components/Office/PPM/PPMHeaderSummary/PPMHeaderSummary.jsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import { number } from 'prop-types';
+import { Label } from '@trussworks/react-uswds';
+import classnames from 'classnames';
+
+import styles from './PPMHeaderSummary.module.scss';
+
+import { PPMShipmentShape } from 'types/shipment';
+import { formatDate, formatCentsTruncateWhole } from 'utils/formatters';
+
+export default function PPMHeaderSummary({ ppmShipment, ppmNumber }) {
+  const {
+    actualPickupPostalCode,
+    actualDestinationPostalCode,
+    actualMoveDate,
+    hasReceivedAdvance,
+    advanceAmountReceived,
+  } = ppmShipment || {};
+
+  return (
+    <header className={classnames(styles.PPMHeaderSummary)}>
+      <div className={styles.header}>
+        <h2>PPM {ppmNumber}</h2>
+        <section>
+          <div>
+            <Label className={styles.headerLabel}>Departure date</Label>
+            <span className={styles.light}>{formatDate(actualMoveDate)}</span>
```

Comment:
The `PPMShipmentShape` expects `actualMoveDate` to be a string, so this might be preferable.

```suggestion
            <span className={styles.light}>{actualMoveDate}</span>
```